R/bootstrap.nlsfit.R

```diff
@@ -471,6 +498,24 @@ bootstrap.nlsfit <- function(fn,
 
   errors <- apply(par.boot[rr, , drop=FALSE], 2, error, na.rm = TRUE)
 
+  if(is.null(priors$param)==FALSE && is.null(priors$p)==FALSE && is.null(priors$psamples)==FALSE){
```

Comment:
This has the same verbose check as before, this can be simplified after the consistency check has been implemented.